Sources/Kitura/FileResourceServer.swift

```diff
@@ -22,6 +22,11 @@ class FileResourceServer {
     /// if file found - send it in response
     func sendIfFound(resource: String, usingResponse response: RouterResponse) {
         guard let resourceFileName = getFilePath(for: resource) else {
+            do {
+                try response.status(.notFound).end()
+            } catch {
+                Log.error("failed to send not found response")
+            }
```

Comment:
~~FileServer should NOT return .notFound when a file is not found. You are preventing other RouterHandlers and Middleware from handling the request.~~

The comments said the change was to FileServer. The was in fact to FileResoureServer. That is a very different story.